library/tinkerpop/src/main/java/uk/gov/gchq/gaffer/tinkerpop/GafferPopGraph.java

```diff
@@ -633,21 +633,22 @@ public <T> T execute(final OperationChain<T> opChain) {
         for (final Operation operation : opChain.getOperations()) {
             operation.setOptions(opOptions);
 
-            if (operation instanceof Input) {
+            if (LOGGER.isDebugEnabled() && operation instanceof Input) {
                 Object input = ((Input) operation).getInput();
                 if (input != null) {
-                    if (input instanceof MappedIterable) {
-                        ((MappedIterable) input).forEach(item -> { LOGGER.info("GafferPop operation input: " + item.toString()); });
+                   if (input instanceof MappedIterable) {
+                        ((MappedIterable) input).forEach(item -> {
+                            LOGGER.debug("GafferPop operation input: {}", item);
+                        });
                     } else {
-                        LOGGER.info("GafferPop operation input: " + input.toString());
+                        LOGGER.debug("GafferPop operation input: {}", input);
                     }
                 }
             }
-
         }
 
         try {
-            LOGGER.info("GafferPop operation chain called: " + opChain.toString());
+            LOGGER.info("GafferPop operation chain called: {}", opChain);
```

Comment:
Would maybe the `toOverviewString()`  method for the operation chain give nicer output here?